Python/Product/VSCode/Publish/Publish.csproj

```diff
@@ -2,16 +2,118 @@
     <PropertyGroup>
         <TargetFramework>netcoreapp2.0</TargetFramework>
     </PropertyGroup>
+    <Import Project="..\..\NetStandard.settings" />
+    <PropertyGroup>
+        <!-- Override build output during dotnet publish so it goes into a separate folder and does not stomp over signed assemblies -->
+        <DotNetPublishPath Condition="'$(DotNetPublishPath)' == ''">$(BuildOutputRoot)publish\</DotNetPublishPath>
+        <!-- Nuget packages must be under the $(OutputPath) folder or signing task will fail. -->
+        <DotNetPublishNugetPath Condition="'$(DotNetPublishNugetPath)' == ''">$(BuildOutputRoot)\raw\nuget\</DotNetPublishNugetPath>
+        <OutputPath Condition="'$(DotNetPublish)' != ''">$(DotNetPublishPath)</OutputPath>
+    </PropertyGroup>
+    <!-- Nuget packaging -->
+    <PropertyGroup>
+        <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+        <NugetPackageVersion>0.1.0</NugetPackageVersion>
```

Comment:
Is this something we should promote to a build property?

Reply:
Yes